server/routes/visitors.test.ts

```diff
@@ -37,21 +37,16 @@ afterEach(() => {
 })
 
 describe('GET /visit/select-visitors', () => {
-  beforeAll(() => {
-    prisonerVisitorsService = new MockPrisonerVisitorsService()
-    sessionApp = appWithAllRoutes(null, null, prisonerVisitorsService, null, systemToken, false, {
-      visitSessionData: {
-        prisoner: {
-          name: 'prisoner name',
-          offenderNo: 'A1234BC',
-          dateOfBirth: '25 May 1988',
-          location: 'location place',
-        },
+  beforeEach(() => {
+    visitSessionData = {
+      prisoner: {
+        name: 'John Smith',
+        offenderNo: 'A1234BC',
+        dateOfBirth: '25 May 1988',
+        location: 'location place',
       },
-    } as SessionData)
-  })
+    }
 
-  it('should render the approved visitor list for offender number A1234BC', () => {
     returnData = {
       prisonerName: 'John Smith',
```

Comment:
I noticed that we've got a bit of duplication with `prisonerName` coming from the visitor list service yet it is now stored in the session. Will tidy this up separately so as to change things in smaller chunks!